grip_api/src/grip_api/task_editor_widgets/socket.py

```diff
@@ -108,32 +124,14 @@ def update_position(self, count_on_side):
         # Change the position of the graphical representation
         self.graphics_socket.setPos(*self.position)
 
-    def add_connector(self, connector):
-        """
-            Add a provided connector to this socket
-
-            @param connector: Connector object to be added to the attribute connectors
-        """
-        self.connectors.append(connector)
-
-    def remove_connector(self, connector):
-        """
-            Remove a provided connector from this socket
-
-            @param connector: Connector object to be removed from the attribute connectors
+    def update_name(self, name):
         """
-        # Make sure it is part of the socket to avoid any exception
-        if connector in self.connectors:
-            self.connectors.remove(connector)
+            Set the name of the socket and update the text displayed by the graphical ToolTip
 
-    def remove_all_connectors(self):
-        """
-            Remove all the connectors associated to this socket
+            @param name: String corresponding to the new name of the socket
         """
-        # Using a while loop because some connectors were not properly deleted with a for loop for an unknown reason
-        while self.connectors:
-            connector = self.connectors.pop(0)
-            connector.remove()
+        self.name = name
```

Comment:
Should be calling the property of the abstract class

Reply:
changed [here](https://github.com/shadow-robot/sr_grip/pull/16/commits/e8ac01c3bac38f78ed47b322574ebdca12c6bd2f)